frontend/src/components/option/utils/SearchBar.tsx

```diff
@@ -49,13 +49,9 @@ function SearchBar({ isExtraOption = false, input, optionList, filterList }: Pro
   };
 
   useEffect(() => {
-    setDropDownList(optionList);
+    input === '' ? setDropDownList([]) : setDropDownList(optionList);
```

Comment:
```suggestion
    !!input ? setDropDownList([]) : setDropDownList(optionList);
```

이건 어떰? !input도 되고! !!연산자는 뒤의 값을 boolean 값으로 평가해줘 :)